api/python/provisioner/commands/set_swupgrade_repo.py

```diff
@@ -191,10 +240,13 @@ def dynamic_validation(self, params: inputs.SWUpgradeRepo, targets: str):  # noq
                     )
                 except KeyError:
                     raise SWUpdateRepoSourceError(
-                        str(repo.source),
-                        f"No release data found in '{RELEASE_INFO_FILE}'"
+                            str(repo.source),
+                            f"No release data found in '{RELEASE_INFO_FILE}'"
                     )
 
+            else:
+                logger.info("Catalog structure validation succeeded")
```

Comment:
that thing should be seemed as a subpart of previous logger.info message which is about the whole stage of repo validation, if you have some quick ideas how to make it clearer please apply.
I see options:
- add `\t` at the beginning
- add some context of the bundle under validation